### ISSUE TITLE
Accept header

### ISSUE THREAD

**Issue by guybedford** [whatwg/loader#29]
Just to bring up the question - is it worth altering the accept header for ES6 module file requests?

**Comment by caridy** [comment]
We have been looking for a fallback mechanism, maybe this is it. E.g.:

```html
<script src="path/to/foo.js" type="module"></script>
```

Where the fallback mechanism can be implemented on the edge by analyzing the headers, and providing the right format.

**Comment by matthewp** [comment]
@caridy what does that have to do with the question of accept headers, can you explain?

**Comment by caridy** [comment]
@matthewp imagine a service that analyzes the header, and infers whether or not the browser supports modules, and based on that it responses with a module or a transpiled version of it. This is not as crazy as it sounds :)

**Comment by matthewp** [comment]
What would be different about the headers? Isn't that @guybedford's question, if the Accept header should be different for modules?

**Comment by caridy** [comment]
Ah, @matthewp, I finally see what you mean, let me rephrase it in my first comment...
